test/common/database.js

```diff
@@ -7,13 +7,12 @@ describe('Database', () => {
   it('should create a table', (done) => {
     let db = Database.create()
 
-    db.connect().then(() => {
-      db.table('test').then(() => { done() })
-    })
+    db.connect().then(() =>
+    db.table('test')).then(() => { done() })
```

Comment:
db.connect().then(return db.table('test')).then(done()) unwraps the call (and chains the promises).